yt_dlp/extractor/gamejolt.py

```diff
@@ -0,0 +1,290 @@
+            return self.url_result(embed_url)
+
+        video_data = traverse_obj(post_data, ('videos', ...), expected_type=dict, get_all=False, default={})
+        formats, thumbnails = [], []
+        for media in video_data.get('media', []):
+            media_url, mimetype, ext, media_id = media['img_url'], media.get('filetype', ''), determine_ext(media['img_url']), media.get('type')
+            if mimetype == 'application/vnd.apple.mpegurl' or ext == 'm3u8':
+                formats.extend(self._extract_m3u8_formats(media_url, post_id, 'mp4', m3u8_id=media_id))
+            elif mimetype == 'application/dash+xml' or ext == 'mpd':
+                formats.extend(self._extract_mpd_formats(media_url, post_id, mpd_id=media_id))
+            elif 'image' in mimetype:
+                thumbnails.append({
+                    'id': media_id,
+                    'url': media_url,
+                    'width': media.get('width'),
+                    'height': media.get('height'),
+                    'filesize': media.get('filesize'),
+                })
+            else:
+                formats.append({
+                    'format_id': media_id,
+                    'url': media_url,
+                    'width': media.get('width'),
+                    'height': media.get('height'),
+                    'filesize': media.get('filesize'),
+                    'acodec': 'none' if 'video-card' in media_url else None,
+                })
+
+        return {
+            'ie_key': GameJoltIE.ie_key(),
+            'extractor': 'GameJolt',
+            'webpage_url': str_or_none(post_data.get('url')) or 'https://gamejolt.com/p/' + post_id,
+            'id': post_id,
+            'title': description,
+            'description': full_description or description,
+            'display_id': post_data.get('slug'),
+            'uploader': user_data.get('display_name') or user_data.get('name'),
+            'uploader_id': user_data.get('username'),
+            'uploader_url': 'https://gamejolt.com' + user_data['url'] if user_data.get('url') else None,
+            'categories': [try_get(category, lambda x: '%s - %s' % (x['community']['name'], x['channel']['display_title']))
+                           for category in post_data.get('communities' or [])],
+            'tags': traverse_obj(
+                lead_content, ('content', ..., 'content', ..., 'marks', ..., 'attrs', 'tag'), expected_type=str_or_none),
+            'like_count': post_data.get('like_count'),
+            'comment_count': post_data.get('comment_count') or 0,
+            'timestamp': int_or_none(post_data.get('added_on'), scale=1000),
+            'release_timestamp': int_or_none(post_data.get('published_on'), scale=1000),
+            'formats': formats,
+            'thumbnails': thumbnails,
+            'view_count': int_or_none(video_data.get('view_count')),
+        }
+
+
+class GameJoltIE(GameJoltBaseIE):
+    _VALID_URL = r'https?://gamejolt\.com/p/(?P<id>[\w-]+)'
```

Comment:
is the site guaranteed to never have `www.` in the url?

Reply:
I swear I thought I already checked this and saw an NXDOMAIN error 😅 Will add